go/client/cmd_id.go

```diff
@@ -33,15 +34,17 @@ func (v *CmdID) ParseArgv(ctx *cli.Context) error {
 	}
 	v.trackStatement = ctx.Bool("track-statement")
 	v.useDelegateUI = ctx.Bool("ui")
+	v.skipCache = ctx.Bool("skip-cache")
```

Comment:
Could we rename this `skip-proof-cache`?  There's another cache for the identify results and don't want callers to get confused about which cache they are skipping...